Source/reactions/Castro_react.cpp

```diff
@@ -248,11 +248,11 @@ Castro::react_state(Real time, Real dt)
 
     // The goal is to update S_old to S_new with the effects of both
     // advection and reactions.  We come into this routine with the
-    // -div{F} stored in hydro_source, and the old and new-time
+    // -div{F} stored together with the time n sources and new-time
```

Comment:
I think this comment should be clarified, it unintentionally sounds like all of these are in Source_Type

Reply:
fixed